manual/pom.xml

```diff
@@ -73,6 +73,20 @@
                     <content>${project.build.directory}/generated-docs</content>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.2.0</version>
```

Comment:
I would not override the version here. It's better to use the version defined in the `<pluginManagement/>` of the parent pom.